<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txt_transcripcion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>801</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="guardar">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>670</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="obtener">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener excel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ver_codigos">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>670</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver códigos de estímulos</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>801</width>
      <height>381</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>799</width>
       <height>379</height>
      </rect>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>941</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Texto extraído del vídeo, puede realizar las correcciones oportunas en el siguiente cuadro de texto:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
